create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/common.py

```diff
@@ -36,6 +36,13 @@
             'port': trafaret.Int(),
         }),
     {%- endif %}
+{%- if cookiecutter.use_sentry == 'y' %}
+    trafaret.Key('sentry'):
```

Comment:
No. Accept a string for dsn.

Reply:
I've put the dsn key here.

Reply:
Either add other options or turn this into a scalar option. No need for mapping if it's just one value.